web/src/components/Landing/Landing.tsx

```diff
@@ -86,17 +90,19 @@ const LandingPage: React.FC<{ collections?: Opds2Feed }> = ({
     </DrbBreakout>
   );
 
-  const contentPrimaryElement = (
+  const contentElement = (
     <Box marginLeft="l" marginRight="l">
-      <Heading level="h2">Recently Added Collections</Heading>
+      <Heading level="h2" marginBottom="s">Recently Added Collections</Heading>
       <CollectionList collections={collections} />
     </Box>
   );
   return (
-    <TemplateAppContainer
-      breakout={breakoutElement}
-      contentPrimary={contentPrimaryElement}
-    />
+    <Template>
+      <TemplateBreakout>{breakoutElement}</TemplateBreakout>
+      <TemplateMain paddingBottom="l">
```

Comment:
I see why this needed to be added, but I'm a little surprised it's not already built in.. @EdwinGuzman, do you know if this is how the component is supposed to work?